index.html

```diff
@@ -1,343 +1,365 @@
-                </a>
-            </li>
-            <li>
-                <a href="www.twitter.com" class="soc-icon">
-                    <img src="Images/Main/Social/Twitter.svg" alt="twitter-mainpage">
-                </a>
-            </li>
-            <li>
-                <a href="https://medium.com/" class="soc-icon">
-                    <img src="Images/Main/Social/Medium.svg" alt="medium-mainpage">
-                </a>
-            </li>
 
+        <ul class="social-icon">
+          <li>
+            <a href="https://github.com/ulises2607" class="soc-icon">
+              <img src="Images/Main/Social/GitHub.svg" alt="github-profile" />
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/cesar-ulises-cappa-subelza/"
+              class="soc-icon"
+            >
+              <img
+                src="Images/Main/Social/Linkedin.svg"
+                alt="linkedin-profile"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.angellist.com/" class="soc-icon">
+              <img src="Images/Main/Social/Angelist.svg" alt="Angelist.com" />
+            </a>
+          </li>
+          <li>
+            <a href="www.twitter.com" class="soc-icon">
+              <img src="Images/Main/Social/Twitter.svg" alt="twitter.com" />
+            </a>
+          </li>
+          <li>
+            <a href="https://medium.com/" class="soc-icon">
+              <img src="Images/Main/Social/Medium.svg" alt="medium.com" />
+            </a>
+          </li>
         </ul>
-        
-
+      </div>
     </section>
 
     <section class="recent-works">
-
-        
-
-        <div class="recent-w-title">
-            <h1>My Recent Works</h1>
-
-            <img src="Images/Work/Desktop/Vector.png" alt="">
-        </div>
-
-
-        <div class="container">
-
-            <img class="img-work" src="Images/Work/Work-img-1.png" alt="Multi-post-stories">
-        
-            <div class="mult-post">
-
-                <h2>Multi-Post Stories</h2>
-
-                <p>
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
-                    has been the industry's standard dummy text ever since the 1500s, when an unknown printer took 
-                    a standard dummy text.
-                </p>
-
-                <ul class="tech">
-                    <li class="tech-btn"><a href="#">CSS</a></li>
-                    <li class="tech-btn"><a href="#">HTML</a></li>
-                    <li class="tech-btn"><a href="#">Bootstrap</a></li>
-                    <li class="tech-btn"><a href="#">Ruby</a></li>
-                </ul>
-
-                <a href="#" class="btn-works-1">See Projects</a>
-            </div>
+      <div class="recent-w-title">
+        <h1>My Recent Works</h1>
+
+        <img src="Images/Work/Desktop/Vector.png" alt="" />
+      </div>
```

Comment:
- [ ] Kindly check my comment on line `46` on the need for the `alt` to be specified with appropriate text 👍